### ISSUE TITLE
False positive on the use of SUDO

### ISSUE THREAD

**Issue by exuperok** [RedCoolBeans/dockerlint#23]
### Issue details

I have a Dockerfile containing the following line:
`RUN echo "jenkins  ALL=(ALL)  ALL" >> etc/sudoers`
which to my knowledge is not running a command with sudo

if i run dockerlint Dockefile, i get the following message 

> WARN:  sudo(8) usage found on line 35 which is discouraged

**Comment by reverbc** [comment]
Same symptom but (slightly) different approach here:
`RUN yum install -y sudo`

This line gets the same warning message and should also be a false positive.

**Comment by iamramin** [comment]
I've just got this false positive with the sudo package listed on a separate line like this:
```
RUN yum install -y \
    sudo
```
